apps/settings/lib/SetupChecks/SupportedDatabase.php

```diff
@@ -38,26 +38,28 @@ public function run(): SetupResult {
 		$version = null;
 		$databasePlatform = $this->connection->getDatabasePlatform();
 		if ($databasePlatform instanceof MySQLPlatform) {
-			$result = $this->connection->prepare("SHOW VARIABLES LIKE 'version';");
-			$result->execute();
+			$statement = $this->connection->prepare("SHOW VARIABLES LIKE 'version';");
+			$result = $statement->execute();
 			$row = $result->fetch();
 			$version = $row['Value'];
 			$versionlc = strtolower($version);
 			// we only care about X.Y not X.Y.Z differences
 			[$major, $minor, ] = explode('.', $versionlc);
 			$versionConcern = $major . '.' . $minor;
 			if (str_contains($versionlc, 'mariadb')) {
-				if (version_compare($versionConcern, '10.3', '<') || version_compare($versionConcern, '10.11', '>')) {
-					return SetupResult::warning($this->l10n->t('MariaDB version "%s" detected. MariaDB >=10.3 and <=10.11 is suggested for best performance, stability and functionality with this version of Nextcloud.', $version));
+				if (version_compare($versionConcern, '10.3', '=')) {
+					return SetupResult::info($this->l10n->t('MariaDB version 10.3 detected, this version is end-of-life and only supported as part of Ubuntu 20.04. MariaDB >=10.6 and <=11.4 is suggested for best performance, stability and functionality with this version of Nextcloud.'));
+				} elseif (version_compare($versionConcern, '10.6', '<') || version_compare($versionConcern, '11.4', '>')) {
+					return SetupResult::warning($this->l10n->t('MariaDB version "%s" detected. MariaDB >=10.6 and <=11.4 is suggested for best performance, stability and functionality with this version of Nextcloud.', $version));
```

Comment:
```suggestion
					return SetupResult::warning($this->l10n->t('MariaDB version "%s" detected. MariaDB >=%s and <=%s is suggested for best performance, stability and functionality with this version of Nextcloud.', $version, 10.6, 11.4));
```
Please format all messages like this so that translators do not need to translate again on each version change.